### PR TITLE
bump version for 8.14

### PR DESCRIPTION
**Pull request by andrei-epure-sonarsource**

### CODE REVIEW

File: sonaranalyzer-dotnet/src/SonarAnalyzer.Vsix/SonarAnalyzer.CSharp.nuspec [hunk 1]
```diff
@@ -12,7 +12,7 @@
     <requireLicenseAcceptance>false</requireLicenseAcceptance>
     <summary>Analyzers that spot Bugs, Vulnerabilities and Code Smells in your code. For an even better overall experience, you can use SonarLint for Visual Studio, which is a free extension that can be used standalone or with SonarQube and/or SonarCloud.</summary>
     <description>Analyzers that spot Bugs, Vulnerabilities and Code Smells in your code. For an even better overall experience, you can use SonarLint for Visual Studio, which is a free extension (https://www.sonarlint.org/visualstudio/) that can be used standalone or with SonarQube (https://www.sonarqube.org/) and/or SonarCloud (https://sonarcloud.io/).</description>
-    <releaseNotes>https://github.com/SonarSource/sonar-dotnet/releases/tag/8.12.0.0</releaseNotes>
+    <releaseNotes>https://github.com/SonarSource/sonar-dotnet/releases/tag/8.14.0.0</releaseNotes>
```
Comment (pavel-mikula-sonarsource) [inline sonaranalyzer-dotnet/src/SonarAnalyzer.Vsix/SonarAnalyzer.CSharp.nuspec:15]: Change from `12` to `14` attracted my attention. Link in this format actually doesn't work without the build number. See
https://github.com/SonarSource/sonar-dotnet/releases/tag/8.13.0.0 404 not found
https://github.com/SonarSource/sonar-dotnet/releases/tag/8.13.0.21683

I'm not sure how to fix this. Will you create a new issue for that?

**Review by pavel-mikula-sonarsource — APPROVED** [review]
LGTM with one comment

File: sonaranalyzer-dotnet/src/SonarAnalyzer.Vsix/SonarAnalyzer.CSharp.nuspec (hunk 1 above)
Comment (andrei-epure-sonarsource) [reply sonaranalyzer-dotnet/src/SonarAnalyzer.Vsix/SonarAnalyzer.CSharp.nuspec:15]: this is done in the ChangeVersion script https://github.com/SonarSource/sonar-dotnet/blob/master/scripts/version/ChangeVersion.proj#L37

the nugets which get published as pipeline artifacts have the correct version inside - you can see the releaseNotes URL on nuget https://www.nuget.org/packages/SonarAnalyzer.CSharp/

Comment (andrei-epure-sonarsource) [reply sonaranalyzer-dotnet/src/SonarAnalyzer.Vsix/SonarAnalyzer.CSharp.nuspec:15]: I looked in the history and I made the change from 8.13 to 8.12 by mistake in this PR https://github.com/SonarSource/sonar-dotnet/commit/8a89c10752604b7b12c43becd25a3737fa0cc3a5#diff-25d0fb1e94395bc61e38cca9d9fb2e36R15

my bad, I'll add one of us as a reviewer on all changes from now on, even if just "documentation" ones like that PR

**Pull request merged**